packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolAncestors.test.ts

```diff
@@ -0,0 +1,54 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { MempoolTx } from '../../../src/category/blockchain'
+import waitForExpect from 'wait-for-expect'
+
+describe('getMempoolAncestors', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+  it('should getMempoolAncestors and return JSON object if verbose is true', async () => {
+    await waitForExpect(async () => {
+      const txId = await client.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+      const mempoolEntry: MempoolTx = await client.blockchain.getMempoolAncestors(txId, true)
```

Comment:
We can remove this redundant typing since it's already typed in https://github.com/DeFiCh/jellyfish/pull/1320/files#diff-401e60eaa7898f5ac9e4d982fcf34c68a2cd450a7a3c372d8243827ba6189c5bR183